client-lib/go/client/jvs_config.go

```diff
@@ -32,8 +32,9 @@ type JVSConfig struct {
 	// Version is the version of the config.
 	Version string `yaml:"version,omitempty" env:"VERSION,overwrite,default=1"`
 
-	// JVS Endpoint. Expected to be fully qualified, including port. ex. http://127.0.0.1:8080
-	JVSEndpoint string `yaml:"endpoint,omitempty" env:"ENDPOINT,overwrite"`
+	// JWKSEndpoint is the full path (including protocol and port) to the JWKS
```

Comment:
I renamed this to more accurately reflect the expected value and updated the comment/example.